src/AppMobile.css

```diff
@@ -0,0 +1,9 @@
+@media (max-width: 769px) {
+	body {
+		font-size: clamp(0.5rem, 2.6vw, 1.5rem);
```

Comment:
ha már az App.css-ben meg van adva a clamp() akkor itt már nem kell, mert a viewport alapján a mobil nézetben már beállítja a font méretet https://css-tricks.com/linearly-scale-font-size-with-css-clamp-based-on-the-viewport/#aa-for-those-who-dont-mind-that-edge-case